src/main_functions_home.c

```diff
@@ -1,110 +1,151 @@
-#include <sys/stat.h>
+#include <stdlib.h>
 #include <stdio.h>
-#include <stdlib.h>	
+#include <stdbool.h>
 #include <sys/types.h>
+#include <sys/stat.h>
 #include <string.h>
-#include <stdbool.h>
 #include <assert.h>
 #include "shellstrings.h"
 #include "parser.h"
 #include "main_functions_home.h"
+#include "log.c/src/log.h"
 
 /* See main_functions_home.h */
-void help_page()
-{
-    shell_help();
-	shell_prompt();
+void help_page(bool *color) {
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!